logistics/scripts/modules/settings/settings-management.psm1

```diff
@@ -0,0 +1,356 @@
+    $populatedSecret = Get-RandomString
+
+    return @{
+        "EdFi.Ods.SandboxAdmin.Web" = @{
+            User = @{
+                Name              = "Test Admin"
+                Email             = "test@ed-fi.org"
+                Password          = Get-RandomString
+                Admin             = "true"
+                NamespacePrefixes = @(
+                    "uri://ed-fi.org"
+                    "uri://gbisd.org"
+                )
+                Sandboxes         = @{
+                    "Populated Demonstration Sandbox" = @{
+                        Key     = $populatedKey
+                        Secret  = $populatedSecret
+                        Type    = "Sample"
+                        Refresh = "true"
+                    }
+                    "Minimal Demonstration Sandbox"   = @{
+                        Key     = Get-RandomString
+                        Secret  = Get-RandomString
+                        Type    = "Minimal"
+                        Refresh = "true"
+                    }
+                }
+            }
+        }
+        "EdFi.Ods.SwaggerUI"        = @{
+            SwaggerUIOptions = @{
+                OAuthConfigObject = @{
+                    ClientId     = $populatedKey
+                    ClientSecret = $populatedSecret
+                }
+            }
+        }
+    }
+}
+
+function Get-ConnectionStringKeyByDatabaseTypes {
+    return @{
+        Ods      = 'EdFi_Ods'
+        Admin    = 'EdFi_Admin'
+        Security = 'EdFi_Security'
+        Master   = 'EdFi_Master'
+    }
+}
+
+function Get-ConnectionStringsByEngine {
+    return  @{
+        SQLServer  = @{
+            ConnectionStrings = @{
+                ((Get-ConnectionStringKeyByDatabaseTypes)['Ods'])      = "Server=(local); Trusted_Connection=True; Database=EdFi_{0};"
+                ((Get-ConnectionStringKeyByDatabaseTypes)['Admin'])    = "Server=(local); Trusted_Connection=True; Database=EdFi_Admin;"
+                ((Get-ConnectionStringKeyByDatabaseTypes)['Security']) = "Server=(local); Trusted_Connection=True; Database=EdFi_Security; Persist Security Info=True;"
+                ((Get-ConnectionStringKeyByDatabaseTypes)['Master'])   = "Server=(local); Trusted_Connection=True; Database=master;"
+            }
+            DatabaseTemplate  = @{
+                MinimalTemplateScript   = 'EdFiMinimalTemplate'
+                PopulatedTemplateScript = 'GrandBend'
+            }
+        }
+        PostgreSQL = @{
+            ConnectionStrings = @{
+                ((Get-ConnectionStringKeyByDatabaseTypes)['Ods'])      = "Host=localhost; Port=5432; Username=postgres; Database=EdFi_{0};"
+                ((Get-ConnectionStringKeyByDatabaseTypes)['Admin'])    = "Host=localhost; Port=5432; Username=postgres; Database=EdFi_Admin;"
+                ((Get-ConnectionStringKeyByDatabaseTypes)['Security']) = "Host=localhost; Port=5432; Username=postgres; Database=EdFi_Security;"
```

Comment:
we may want to use environment vars for host, port and username in the future. thoughts?